src/containers/org/MemberRoleCard.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import React from 'react';
+
+import styled from 'styled-components';
+import { faTimes } from '@fortawesome/pro-light-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Colors, IconButton } from 'lattice-ui-kit';
+import { Link } from 'react-router-dom';
+import type { Role, UUID } from 'lattice';
+
+import { RoleIcon } from '../../assets/svg/icons';
+import { ORG_ID_PARAM, ORG_ROLE, ROLE_ID_PARAM } from '../../core/router/Routes';
+
+const { NEUTRAL } = Colors;
+
+const RoleCard = styled.div`
+  align-items: center;
+  background-color: ${NEUTRAL.N50};
+  border-radius: 4px;
+  border: 1px solid ${NEUTRAL.N50};
+  display: grid;
+  grid-gap: 16px;
+  grid-template-columns: 1fr auto;
+  justify-content: space-between;
+  padding: 8px 24px;
+`;
```

Comment:
No change required, just some thoughts. A copy of this exists in `DataSetPermissionsCard`. Perhaps we can consolidate?